include/boost/geometry/algorithms/frechet_distance.hpp

```diff
@@ -0,0 +1,177 @@
+/*  
```

Comment:
you transform your copyright to something similar to this https://github.com/BoostGSoC18/geometry/blob/78411f63c8d552189c34bb3c5589696c9c4f6c8a/include/boost/geometry/formulas/karney_direct.hpp#L1(only first 9 lines apply to you)

Reply:
Ok, I will update the copyright.